web/src/utils/state/atoms.ts

```diff
@@ -67,16 +67,18 @@ export const feedbackCardCollapsedNumberAtom = atom(0);
 
 export const colorblindModeAtom = atomWithStorage('colorblindModeEnabled', false);
 
-export const dataSourcesCollapsedBarBreakdown = atom(true);
+export const dataSourcesCollapsedBarBreakdown = atom<boolean>(true);
 
-export const dataSourcesCollapsedBreakdown = atom(true);
+export const dataSourcesCollapsedBreakdown = atom<boolean>(true);
 
-export const dataSourcesCollapsedEmission = atom(true);
+export const dataSourcesCollapsedEmission = atom<boolean>(true);
 
-export const userLocationAtom = atom(undefined);
+export const userLocationAtom = atom<string | undefined>(undefined);
 
 export const hasSeenSurveyCardAtom = atomWithStorage('hasSeenSurveyCard', false);
 
+export const hasSeenUsSurveyCardAtom = atomWithStorage('hasSeenUsSurveyCard', false);
```

Comment:
But not here?

Reply:
The cognitive load required was higher 🤷